Add Market exchange subclasses #677

stap-m opened the pull request:
closes #677 

I'd like to move `licence` from oeo-model to oeo-shared, thus `emission certificate` (subclass of licence) can be part of oeo-social. Is it possible to move the class without deleting and recreating it @sfluegel05 ?

sfluegel05 commented:
I think the easiest way is to just cut and paste the class with a text editor.

stap-m commented:
I also moved `emission quantity value` to oeo-shared, but sth went wrong. I can't open oeo-physical in protege and the CI check gives an error. I can't find out why. @sfluegel05 any clue?

`is about` isn't part of oeo-social either. I'm unsure how to import it there: the IAO-module is importet, but not this relation?! And is there a reason, why we don't import it to oeo-shared directly?

sfluegel05 commented:
> I also moved `emission quantity value` to oeo-shared, but sth went wrong. I can't open oeo-physical in protege and the CI check gives an error. I can't find out why. @sfluegel05 any clue?

I don't exactly why this is necessary, but you have to declare the class `emission quantity value` in oeo-physical if you want to use it in oeo-physical, even if you import it from another file. Protege does this automatically as soon as someone starts using an imported class, but here we have to do this manually.

> `is about` isn't part of oeo-social either. I'm unsure how to import it there: the IAO-module is importet, but not this relation?! And is there a reason, why we don't import it to oeo-shared directly?

after merging PR #710, the iao-module gets imported into oeo-shared, those changes were just missing on this branch.

Now everything should be working again as intended.

stap-m commented:
Looks great, thanks @sfluegel05 
I updated changelog and added the relation `emission quantity value` `quantity value of` `some` `emission certificate`. Now it should be ready.

l-emele reviewed (changes requested):
Only a minor thing: Could you please add EUA as alternative term to `EU Allowance`?
